config.py

```diff
@@ -115,7 +115,7 @@ class Config:
 
     FRONTEND_ADDRESS = 'https://dm-dev.apps.staging.digital.gov.au'
     ADMIN_ADDRESS = 'https://dm-dev-admin.apps.staging.digital.gov.au'
-    REACT_APP_ROOT = '/2'
+    APP_ROOT = {'digital-marketplace': '/2', 'orams': '/orams'}
```

Comment:
shouldn't we work this off slug e.g. `'dm'`

Reply:
framework slug is `digital-marketplace`, we can change to `dm`?

Reply:
you're right